Add meta description to some pages (#4566)
wazuh/wazuh-documentation: source/amazon/services/supported-services/vpc.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: Amazon Virtual Private Cloud (Amazon VPC) lets users provision a logically isolated section of the AWS Cloud where they can launch AWS resources in a virtual network that they define. 

Reviewer: ```suggestion
  :description: Amazon Virtual Private Cloud lets users provision a logically isolated section of the AWS Cloud to launch AWS resources in a virtual network.
```
